### PR TITLE
updates for redhat scans

### PR DESCRIPTION
**Pull request by ryandawsonuk**
fixes https://github.com/SeldonIO/seldon-core/issues/2869

Been a change to RH scanning process. New yum update required to pass vulnerability scanning.

Also noticed one of the images was ubi7, which I think won't work next time we try it. 

Putting these updates in place before our next release.

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **ryandawsonuk**
You can assign the PR to them by writing `/assign @ryandawsonuk` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Fri Jan 22 11:48:05 UTC 2021
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=1

**Comment by seldondev** [comment]
Fri Jan 22 11:48:05 UTC 2021
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=2

**Comment by axsaucedo** [comment]
@ryandawsonuk the engine image is being depricated - will it still be required to keep up to date with all the vulnerability checks?

**Comment by seldondev** [comment]
Fri Jan 22 17:34:18 UTC 2021
The logs for [pr-build] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=3

**Comment by seldondev** [comment]
Fri Jan 22 17:34:20 UTC 2021
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=4

**Comment by ryandawsonuk** [comment]
Should not be necessary to keep scanning engine going forward. Only including it here in case we publish it with 1.6.0.

**Comment by seldondev** [comment]
Mon Jan 25 14:47:12 UTC 2021
The logs for [lint] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=6

**Comment by seldondev** [comment]
Mon Jan 25 14:47:15 UTC 2021
The logs for [pr-build] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=5

**Comment by seldondev** [comment]
Mon Jan 25 15:19:36 UTC 2021
The logs for [pr-build] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=7

**Comment by seldondev** [comment]
Mon Jan 25 15:19:55 UTC 2021
The logs for [lint] [8] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/8.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=8

File: components/seldon-request-logger/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM registry.access.redhat.com/ubi8/python-36
+FROM registry.access.redhat.com/ubi8/python-38
```
Comment (ryandawsonuk) [inline components/seldon-request-logger/Dockerfile:1]: I published this for 1.5.1. Had to change this base image for it to pass.

**Comment by seldondev** [comment]
Tue Jan 26 14:47:57 UTC 2021
The logs for [pr-build] [9] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/9.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=9

**Comment by seldondev** [comment]
Tue Jan 26 14:47:59 UTC 2021
The logs for [lint] [10] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/10.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=10

**Comment by seldondev** [comment]
Tue Jan 26 14:53:55 UTC 2021
The logs for [lint] [13] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/13.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=13

**Comment by seldondev** [comment]
Tue Jan 26 14:53:54 UTC 2021
The logs for [pr-build] [11] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/11.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=11

**Comment by seldondev** [comment]
Tue Jan 26 14:54:00 UTC 2021
The logs for [integration] [12] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/12.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=12

**Comment by seldondev** [comment]
@ryandawsonuk: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
pr-build | d5a5cb9a92422e145a79ccbc544957fc64f888fc | [link]() | `/test this`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@ryandawsonuk: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | d5a5cb9a92422e145a79ccbc544957fc64f888fc | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Wed Jan 27 14:50:38 UTC 2021
The logs for [integration] [14] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/14.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=14

**Comment by seldondev** [comment]
@ryandawsonuk: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | d5a5cb9a92422e145a79ccbc544957fc64f888fc | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Wed Jan 27 18:08:34 UTC 2021
The logs for [integration] [15] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/15.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=15

**Comment by seldondev** [comment]
@ryandawsonuk: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | d5a5cb9a92422e145a79ccbc544957fc64f888fc | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by ryandawsonuk** [comment]
Getting a different failure with each re-run.

Failed test in 15.log: TestTagsPythonS2iK8s.test_model_combiner_grpc
Failed test in 14.log: test-label-update-1-4-0
Failed test in 12.log: TestPrepack.test_text_alibi_explainer

**Comment by seldondev** [comment]
Thu Jan 28 09:53:32 UTC 2021
The logs for [integration] [16] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/16.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=16

**Comment by seldondev** [comment]
@ryandawsonuk: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | d5a5cb9a92422e145a79ccbc544957fc64f888fc | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
Thu Jan 28 10:10:14 UTC 2021
The logs for [integration] [17] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2870/17.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2870 --build=17

**Comment by axsaucedo** [comment]
@ryandawsonuk looks good, good to go or still needs to run notebook tests?

**Comment by axsaucedo** [comment]
Merging as discussed with @ryandawsonuk

**Pull request merged**